src/apps/pillarx-app/api/homeFeed.ts

```diff
@@ -17,9 +21,29 @@ export const homeFeedApi = createApi({
     })
 })
 
+// maxRetries: 5 is the default, and can be omitted. Shown for documentation purposes.
+const staggeredBaseQuery = retry(fetchBaseQuery({baseUrl:
+    process.env.REACT_APP_USE_TESTNETS === 'true' ?
+    'https://walletportfolio-nubpgwxpiq-uc.a.run.app' :
+    'https://walletportfolio-7eu4izffpa-uc.a.run.app' }), {
+    maxRetries: 5,
+  })
+
+export const walletTiledApi = createApi({
+    reducerPath: 'walletTiledApi',
```

Comment:
`walletTiledApi` seems like a spelling mistake, is it supposed to be

```suggestion
export const walletPortfolioTileApi = createApi({
    reducerPath: 'walletPortfolioTileApi',
```